### ISSUE TITLE
Files duplicated in file_elab.lst 

### ISSUE THREAD

**Issue by nmoroze** [chipsalliance/Surelog#2200]
I noticed that if I parse a file like so: `surelog -parse test.v`, duplicate entries get added to `slpp_all/file_elab.lst`, where one doesn't include `slpp_all/`:
```
./slpp_all/work/_6142509188972423790/test.v
work/_6142509188972423790/test.v
```

This appears to have changed between 5b07b2 and 17a2f48: if I test this on 5b07b2 I see only one entry in file_elab.lst.

**Comment by alaindargelas** [comment]
Strange...

**Comment by alaindargelas** [comment]
Fixed by #2626. A new file map list I introduced hastily created that dupplication

**Comment by nmoroze** [comment]
Thanks for the quick fix!
